airship/kubeadm-airship-gap-analysis.md

```diff
@@ -25,6 +25,79 @@ Jira issues:
 Jira issues:
 - [Non-default IP for control plane components](https://airship.atlassian.net/browse/AIR-146)
 
+## Motivation
+For this experiment our motivation is to know the answers to the following questions
```

Comment:
add the following at the end:
"regarding control plane components"

Reply:
done